scene/resources/resource_format_text.h

```diff
@@ -54,6 +54,7 @@ class ResourceLoaderText {
 	};
 
 	bool is_scene = false;
+	int format_version;
```

Comment:
Needs to be initialized to some value (especially when `format` field is optional).

Reply:
I meant initialized here :P
Same with `use_compat`.